contrib/junit.tpl

```diff
@@ -15,6 +15,16 @@
     {{- end }}
     </testsuite>
 
+{{- if .Licenses }}
```

Comment:
Should this block be placed after Misconfigs?

Reply:
Sure, I don't think it matters. Since the license scanner is not active by default, I assume that users who want to use the license scanner (especially with the JUnit template) will select the scanners accordingly, and probably run specifically only the license scanner in that instance.

It's possible of course to get a JUnit report with all the scanners' results in a single XML file, but since vulnerabilities and license compliance have different impact and scope, that would be somewhat unwieldy to handle. But yes, in that case, the order of testsuites might matter. I think it's fine to move it to after the misconfigs.

If you agree, I can update my commits.

Reply:
I agree. 
actually it doesn't matter, but let's move it. thanks!